src/Forms/ImportPreviewForm/GroupPreview/GroupPreview.jsx

```diff
@@ -57,8 +62,7 @@ const GroupPreview = ({
 
   useEffect(() => {
     fetchPackages();
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, []);
+  }, [fetchPackages]);
```

Comment:
is this change intentionally? This means that if the callback changes it's refetched again.

Reply:
This might have been done by my litter after I have deleted this line unintentionally while coding/testing. I will fix this later.